setup.py

```diff
@@ -123,9 +123,9 @@
         "Intended Audience :: Education",
         "Intended Audience :: Science/Research",
         "Operating System :: OS Independent",
-        "Programming Language :: Python :: 3.7",
-        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3",
         "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
```

Comment:
Why not 3.11?

Reply:
added in https://github.com/huggingface/optimum/pull/2086/commits/44059cc1fa785ff96fbc01be0be00d64fc55e136